apps/rest-api-server/api/index.ts

```diff
@@ -1,34 +1,46 @@
 import bodyParser from "body-parser";
 import cors from "cors";
 import express from "express";
-import { collectDefaultMetrics, register } from "prom-client";
 import swaggerUi from "swagger-ui-express";
 import { createOpenApiExpressMiddleware } from "trpc-openapi";
 
-import { appRouter, createTRPCContext } from "@blobscan/api";
+import {
+  appRouter,
+  createTRPCContext,
+  getPrismaMetricsClient,
+} from "@blobscan/api";
+import { collectDefaultMetrics, promRegister } from "@blobscan/open-telemetry";
 
 import { env } from "./env";
 import { logger } from "./logger";
 import { morganMiddleware } from "./middlewares/morgan";
 import { openApiDocument } from "./openapi";
 
-collectDefaultMetrics();
-
 const app = express();
 
 app.use(cors());
 app.use(bodyParser.json({ limit: "2mb" }));
 app.use(morganMiddleware);
 
-// eslint-disable-next-line @typescript-eslint/no-misused-promises
-app.get("/metrics", async (_req, res) => {
-  try {
-    res.set("Content-Type", register.contentType);
-    res.end(await register.metrics());
-  } catch (err) {
-    res.status(500).end(err);
-  }
-});
+if (!env.OTEL_SDK_DISABLED) {
+  collectDefaultMetrics();
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  app.get("/metrics", async (_req, res) => {
```

Comment:
Yeah, I'm not a big fan of this variable either, but I wanted to reuse as many already-defined open telemetry env variables as possible to avoid adding new redundant ones.

If it's more legible to have the "_ENABLED"  variable I can add it